### DISCUSSION TITLE
Ran into small issue with a prefix for Get-Metadata for the first time.

### DISCUSSION THREAD

**Discussion by codypaternostro** [MilestoneSystemsInc/PowerShellSamples#150] [Ideas]
I was reading about the [VMS command prefix ](https://www.milestonepstools.com/getting-started/#why-are-some-commands-prefixed-with-vms) after having my first conflict with _Get-Metadata_. I wanted to install a [configuration module ](https://github.com/PoshCode/Configuration/pulls?q=is%3Aclosed) that had the same command _Get-Metadata_. As a simple solution, I could prefix the command myself and move on, but that might cause issues down the line. 

I wanted to suggest a idea that the current set of commands use the [appdata path for connection profiles](https://www.milestonepstools.com/blog/2023/09/29/introducing-connection-profiles-and-custom-attributes/#how-do-they-work) to possibly store the history of the user's current list of commands as a point of reference. Then, a module-wide switch/boolean value set could detect commands not using the VMS prefix and modify them locally. If a user ever wanted to switch back, they could either reinstall the module or use the saved history to restore to the original.

This idea may be too intense, but an alternative solution may exist. Please share what you think!
![get-metadata](https://github.com/user-attachments/assets/c09ea7c1-a3a9-4773-9b46-f99d7d6bb663)

**Comment by joshooaj** [comment]
FYI - I'm looking to rename/remove commands/aliases with name collisions in an upcoming release https://github.com/MilestoneSystemsInc/PowerShellSamples/discussions/171
